jdaviz/configs/default/plugins/gaussian_smooth/gaussian_smooth.py

```diff
@@ -19,7 +19,13 @@
 u.add_enabled_units([spaxel])
 
 
-@tray_registry('g-gaussian-smooth', label="Gaussian Smooth")
+@tray_registry(
+    'g-gaussian-smooth', label="Gaussian Smooth",
+    viewer_reference_name_kwargs={
+        "_default_spectrum_viewer_reference_name":
+            ["spectrum_viewer_reference_name", {"require_spectrum_viewer": True}],
+    }
+)
```

Comment:
Most recently in [cd3aee7](https://github.com/spacetelescope/jdaviz/pull/1681/commits/cd3aee73f412cd701cd69b7e9a79cb405ed76b0d) I address a comment from @kecnry via Slack that it'd be more sustainable to add the plugin constructor keyword arguments to the plugin machinery (rather than as a private dictionary within `app.py`). My solution is to add the requirements for the plugin to the `tray_registry` decorator. This way, future plugin authors can configure the requirements for their plugins.